optaplanner-docs/src/modules/ROOT/pages/use-cases-and-examples/vaccination-scheduling/vaccination-scheduling.adoc

```diff
@@ -0,0 +1,266 @@
+[id="chap-optaplanner-vaccination_{context}"]
+= OptaPlanner Vaccination Scheduler
+
+:OPTAPLANNER-COMM:
+//Variable for the community version conditional statements.
+//:OPTAPLANNER-ENT:
+//Variable for the enterprise version conditional statements.
+
+ifdef::OPTAPLANNER-COMM[]
+
+:imagesdir: ../..
+
+:OPTAPLANNER: OptaPlanner
+:QUARKUS: Quarkus
+:OPTAPLANNER_JDK_VERSION: 11
+:OPTAPLANNER_MAVEN_VERSION: 3.6
```

Comment:
these come from the build
The attributes.adoc file should have a comment that these properties come from the build and exist.
Optionally, the attributes.adoc file can default them to "<ERROR - optaplanner-maven-version not set>"

If it's the minimum maven version, then it should contain minimum (or min).